presto-docs/src/main/sphinx/release/release-327.rst

```diff
@@ -0,0 +1,68 @@
+Release 327 (16 Dec 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix out-of-order data source reads when using ``work_processor_pipelines``. (:issue:`2144`)
+* Fix failure of :func:`word_stem` for certain inputs. (:issue:`2145`)
+* Fix potential incorrect results for queries involving ``FILTER (WHERE ...)`` when the condition
+  is a reference to a table column. (:issue:`2267`)
+* Add rename view. (:issue:`1060`)
+* Add execution time statistics for event listeners. (:issue:`2247`)
+* Add ``error_type`` and ``error_code`` columns to ``system.runtime.queries``. (:issue:`2249`)
+* Rename ``experimental.work-processor-pipelines`` configuration property to ``experimental.late-materialization.enabled``
+  and rename ``work_processor_pipelines`` session property to ``late_materialization``. (:issue:`2275`)
+
+Security Changes
+----------------
+
+* Add support for configuration of multiple system access controls. (:issue:`2178`)
+
+Hive Connector Changes
+----------------------
+
+* Fix incorrect query results when reading ``timestamp`` values from ORC files written by Hive 3.1 or later. (:issue:`2099`)
+* Fix a CDH 5.x metastore compatibility issue resulting in failure when inserting data into, or
+  analyzing a table with ``date`` columns. (:issue:`556`)
+* Reduce number of metastore calls when fetching partitions. (:issue:`1921`)
+* Support reading from insert-only transactional tables. (:issue:`576`)
+* Deprecate ``parquet.fail-on-corrupted-statistics`` (previously known as ``hive.parquet.fail-on-corrupted-statistics``).
+  Setting this configuration property to ``false`` may hide correctness issues, leading to incorrect query results.
+  Session property ``parquet_fail_with_corrupted_statistics`` is deprecated as well.
+  Both configuration and session properties will be removed in a future version. (:issue:`2129`)
+* Improve concurrency when updating table or partition statistics. (:issue:`2154`)
+* Add rename view. (:issue:`2189`)
+* Add ``orc_use_column_names`` hive session property. (:issue:`2248`)
+
+
+Kudu Connector Changes
+----------------------
+
+* Support predicate push down in for decimal type. (:issue:`2131`)
+* Fix accidental column position swap. (:issue:`2252`)
+* Improve predicate pushdown by using in-list Kudu predicate of ``IN`` Presto operator.
+  This improves queries like: ``SELECT * FROM kudu_table WHERE str_column IN ( 'hello', 'world')``. (:issue:`2253`)
+
+MongoDB Connector Changes
+-------------------------
+
+* Add support for accessing a view. (:issue:`2156`)
+
+PostgreSQL Connector Changes
+----------------------------
+
+* Allow to convert unsupported types to ``VARCHAR`` with catalog session property
+  ``unsupported_type_handling= CONVERT_TO_VARCHAR`` or connector configuration property
+  ``unsupported-type-handling=CONVERT_TO_VARCHAR``. (:issue:`1182`)
+
+MySQL Connector Changes
+-----------------------
+
+* Fix query failure when ``GTID`` mode is enabled. (:issue:`2251`)
```

Comment:
```
* Fix failure of ``CREATE TABLE AS`` when ``GTID`` mode is enabled.

Reply:
I think this was about `INSERT` actually. I know https://github.com/prestosql/presto/issues/2251 initially stated CREATE TABLE, but the stacktrace was for INSERT.
And I guess CTAS was _not_ affected.
@ebyhr ?

Reply:
Yes, only INSERT statement was a problem (CTAS was fine). I confirmed again just in case.